### PR TITLE
string-manipulation-3 Challenge - Sang Tran

### PR DESCRIPTION
**Pull request by Stran1025**

### CODE REVIEW

**Comment by Stran1025** [comment]
screencapture-127-0-0-1-5500-string-manipulation-3-index-html-2022-03-30-12_52_56

File: string-manipulation-3/src/is-anagram.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+*/
+
+function isAnagram(string1, string2) {
+  var string1Lowercase = string1.toLowerCase();
+  var array1 = string1Lowercase.split('');
+  var string2Lowercase = string2.toLowerCase();
+  var array2 = string2Lowercase.split('');
+  for (var i = array1.length - 1; i >= 0; i--) {
+    if (array1[i] === ' ') {
+      array1.pop();
```
Comment (codingcodymiller) [inline string-manipulation-3/src/is-anagram.js:28]: Why `pop`? Wouldn't you want to remove this specific space character?

File: string-manipulation-3/src/title-case.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+function titleCase(string) {
+  var output = '';
+  var word = '';
+  var lowercase = string.toLowerCase();
+  var array = lowercase.split(' ');
+  var doNotCapitalize = ['and', 'or', 'nor', 'but', 'a', 'an', 'the', 'as', 'at', 'by', 'for', 'in', 'of', 'on', 'per', 'to'];
+  for (var i = 0; i < array.length; i++) {
+    if (i === 0) {
+      if (array[i] === 'javascript:') {
```
Comment (codingcodymiller) [inline string-manipulation-3/src/title-case.js:38]: Why does this check only happen for the first word? What if the title was "Adventures in JavaScript: Living By A Code"?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]
Nice work!!

**Pull request merged**